data/trello_items.py

```diff
@@ -0,0 +1,82 @@
+from data.todo_item import Item
```

Comment:
This file looks like a nice way to contain some of the functionality we might need to communicate with Trello, but it looks to me like these functions aren't actually used currently - if that's right it might be clearest to remove this file (we can always recover it from source control) or update your routes in app.py to use these methods as appropriate